### PR TITLE
E2E Test Cases Update: add kafka, zipkin

### PR DESCRIPTION
**Pull request by sakows**
Add Kafka Test Case( Headers, Src_uid, Response_Time) Issue #37
Add Zipkin Test Case
Update configuration file

### CODE REVIEW

File: istio/release/install/zipkin/zipkin.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (sehz) [inline istio/release/install/zipkin/zipkin.yaml]: this is duplicate of Istio deployment. no need to copy here

File: istio/tests/cases/10_kafka_messages_spec.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+with description('Testing Kafka messages'):
+    with before.all:
+         #Read Config file
+         configuration.setenv(self)
+         configuration.generate_request(self)
+
+with context('Starting test'):
+    with it('Testing Kafka functionality'):
+
+        proc_a = configuration.mp.Process(target=configuration.check_kafka_logs,args=(self,configuration.kafka_mp_output))
```
Comment (sehz) [inline istio/tests/cases/10_kafka_messages_spec.py]: add actual test scenario

File: istio/tests/cases/11_zipkin_spec.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+headers = {'content-type': 'application/json','accept': 'application/json'}
+with description('Zipkin tracing functionality'):
+    with before.all:
+         #Read Config file
+         configuration.setenv(self)
+
+    with context('Set environment'):
+         with it('Add Zipkin tracing feature'):
+            configuration.run_shell("kubectl apply -f "+configuration.nginmesh_install_path+"zipkin/zipkin.yaml","run")
```
Comment (sehz) [inline istio/tests/cases/11_zipkin_spec.py]: assume zipkin is installed so no need to deploy everytime

File: istio/tests/cases/11_zipkin_spec.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            expect(r1.status_code).to(equal(200))
+            if 'productpage' in r1.text:
+                expect(0).to(equal(0))
+            else:
+                expect(0).not_to(equal(0))
+            configuration.generate_request(self)
+
+    with context('Clean Environment'):
+        with it('Delete Zipkin tracing feature'):
+            configuration.run_shell("kubectl delete -f "+configuration.nginmesh_install_path+"zipkin/zipkin.yaml","run")
```
Comment (sehz) [inline istio/tests/cases/11_zipkin_spec.py]: no need to clean zipkin

File: istio/tests/cases/configuration.py [hunk 5]
```diff
@@ -23,16 +36,60 @@ def run_shell(self,type):
...
+                user_agent=count_init
+                req_path=count_init
+                response_code=count_init
+                response_duration=count_init
+                source_uid=count_init
+                content_type=count_init
+                trace_id=count_init
+                span_id=count_init
+
+                if kafka_mock_hdr.get('x-request-id') in out:
```
Comment (sehz) [inline istio/tests/cases/configuration.py]: verify each header to ensure it's right value

File: istio/tests/cases/10_kafka_messages_spec.py (hunk 2 above)
Comment (sakows) [reply istio/tests/cases/10_kafka_messages_spec.py]: ok

File: istio/tests/cases/configuration.py (hunk 5 above)
Comment (sakows) [reply istio/tests/cases/configuration.py]: ok

File: istio/tests/cases/.coverage [hunk 6]
```diff
@@ -1 +0,0 @@
...on2.7/site-packages/gevent/monkey.py":[523,88,686,534,537,548,549,552,564,53,54,55,568,569,570,59,60,61,62,63,576,577,66,67,580,581,582,72,73,74,76,80,84,85,86,599,600,92,606,607,608,609,610,187,100,614,617,616,105,618,107,620,621,622,111,624,625,626,627,116,629,119,635,106,640,129,130,131,133,136,137,138,139,140,143,110,151,152,153,154,155,156,157,159,160,161,164,113,683,684,685,174,175,176,108,115,180,696,697,699,701,109,213,227,230,232,233,234,237,637,275,276,277,95,567,56,114,575,64,65,68,69,70,427,432,628,438,439,442,443,444,448,458,463,466],"/Users/shahriyarmammadov/.local/share/virtualenvs/cases-RdUQzwPy/lib/python2.7/site-packages/urllib3/util/connection.py":[1,2,3,4,7,405,393,394,395,396,130,398,399,275,276,277,985,36,37,988,50,51,53,184,58,60,652,76,88,89,986,655,92,93,1125,96,1122,1123,101,102,103,104,1130,107,1132,109,110,112,510,118,119,120,121,125,126,127],"/Users/shahriyarmammadov/.local/share/virtualenvs/cases-RdUQzwPy/lib/python2.7/site-packages/chardet/sbcsgroupprober.py":[34,35,37,38,39,40,43,44,29,30,31],"/Users/shahriyarmammadov/.local/share/virtualenvs/cases-RdUQzwPy/lib/python2.7/site-packages/chardet/langhebrewmodel.py":[193,194,195,196,197,198,199,53,190]}}
```
Comment (sehz) [inline istio/tests/cases/.coverage:1]: remove this if not needed

File: istio/tests/cases/10_kafka_messages_spec.py [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                user_agent=0
+                req_path=0
+                response_code=0
+                response_duration=0
+                source_uid=0
+                content_type=0
+                trace_id=0
+                span_id=0
+
+                if kafka_mock_hdr.get('x-request-id') in out:
```
Comment (sehz) [inline istio/tests/cases/10_kafka_messages_spec.py:32]: do assertion for each value

**Pull request closed without merging**